### DISCUSSION TITLE
Protection /rt

### DISCUSSION THREAD

**Discussion by Diezel0468** [Zrips/CMI#5951] [Enhancement]
Please add protection for the /rt command. What do I mean by protection? To prevent the player from entering someone else's private region during the /rt command. For example: turrets are installed on the server and when a player makes a random teleportation, he is immediately killed by turrets in a foreign private region.

**Comment by CloudeLecaw** [comment]
Well this is normaly not the Problem of cmi to keep track of your regions

**Comment by Diezel0468** [comment]
> Well this is normaly not the Problem of cmi to keep track of your regions

Well, let's say the betterrtp plugin has region tracking, why not add this function here? Plus, you could add a border tracking function. So that rtp was only within the radius of the barrier of the world.

**Comment by CloudeLecaw** [comment]
Cmi doesnt have and betterrpt also just jave for worldgaurd what isnt used by many anymore
